src/main/java/hudson/plugins/jira/pipeline/IssueWorkflowActionStep.java

```diff
@@ -0,0 +1,115 @@
+package hudson.plugins.jira.pipeline;
+
+import com.google.inject.Inject;
+import hudson.Extension;
+import hudson.Util;
+import hudson.model.Result;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.plugins.jira.JiraSite;
+import hudson.plugins.jira.Messages;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution;
+import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Created by aleksandr on 01.07.16.
+ */
+public class IssueWorkflowActionStep  extends AbstractStepImpl {
+    private final String issueKey;
```

Comment:
As there is no variable expansion here, I'm wondering what usecase is this going to be, because you'll need to configure "issueKey" as static value in the plugin configuration.
Usually this will not be the case and you'll want to get this from e.g. a variable ?

Maybe you could add variable expansion (see *JiraIssueUpdateBuilder* for example) - that would give some additional great power to those functionalities, what do you think?